roles/satellite-clone/tasks/restore.yml

```diff
@@ -6,10 +6,10 @@
   service: name=postgresql state=stopped
 
 - name: Restore postgresql data
-  command: tar --selinux --overwrite -xf {{ backup_dir }}/pgsql_data.tar.gz -C /
+  command: "tar --selinux --overwrite -xf {{ backup_dir }}/pgsql_data.tar.gz -C /"
```

Comment:
what is the reason behind including double quotes here?

Reply:
It was part of debugging a different issue (that was solved by the delete `cp_done` task) but I think we actually should be using quotes wherever string interpolation `{{ string }}` is used